### PR TITLE
Migrate the JupyterLab preview extension to JLab2

### PR DESCRIPTION
**Pull request by echarles**
This PR intent is to migrate the JupyterLab preview extension to JLab2.

For now the code builds, I still need to check if the functionality is not degraded.

### CODE REVIEW

**Comment by echarles** [comment]
@jtpio This is working for me on JLab2.

Screenshot 2020-03-12 at 06 13 13

**Comment by jtpio** [comment]
Thanks @echarles, this looks good!

File: packages/jupyterlab-voila/src/index.ts [hunk 1]
```diff
@@ -66,7 +68,7 @@ class VoilaRenderButton
     const button = new ToolbarButton({
       className: "voilaRender",
       tooltip: "Render with Voila",
-      iconClassName: VOILA_ICON_CLASS,
+      iconClass: VOILA_ICON_CLASS,
```
Comment (jtpio) [inline packages/jupyterlab-voila/src/index.ts]: Good thing the icons still support this :+1: 

We could update to `LabIcon` at a later stage (in a separate PR).

Comment (echarles) [reply packages/jupyterlab-voila/src/index.ts]: Yeah I have been suprised it was still working fine with that. Upgrading to `LabIcon` also popped up in my mind for another PR.

**Comment by jtpio** [comment]
Works well locally :+1: 

preview-lab-2

File: packages/jupyterlab-voila/src/index.ts (hunk 1 above)
Comment (jtpio) [reply packages/jupyterlab-voila/src/index.ts]: It was fixed in https://github.com/jupyterlab/jupyterlab/pull/7947

**Pull request merged**